src/components/Home.js

```diff
@@ -17,6 +18,7 @@ const Home = (props) => {
         pokemons={pokemons}
         searchedName={searchedName}
         loading={loading}
+        searchPokemon={new SearchPokemon()}
```

Comment:
This is a simple way to inject the dependency to the component. This way we can replace the behavior when we need.